### PR TITLE
feat(server): Polkassembly chain-db server

### PR DESCRIPTION
**Pull request by Tbaut**
closes https://github.com/paritytech/Nomidot/issues/118
closes https://github.com/paritytech/polkassembly/issues/290

### CODE REVIEW

**Comment by yjkimjunior** [comment]
should this be in Polkassembly?

**Comment by Tbaut** [comment]
To me, it sits right in front of Prisma. I don't mind moving it elsewhere, but I really think it's a good idea to have your "server" (effectively nomidot front server) close to this one. But you decide.

**Comment by Tbaut** [comment]
How did you get around the no explicit any @yjkimjunior ?

**Comment by yjkimjunior** [comment]
> How did you get around the no explicit any lint errors @yjkimjunior ?

could just create a type? I think for some of this boilerplate stuff, e.g. `parent` it's all typed as `any` by Apollo themselves so I wouldn't waste too much time with that.

**Comment by Tbaut** [comment]
> hmm that makes sense too. I would personally prefer it to be in Polkassembly just for separation of concerns but if this is easier for you, I don't mind too much for now

Sounds good, then let's keep this here for now and we can still move it any time it it disturbs, it's a matter of copy/pasting.

> could just create a type? I think for some of this boilerplate stuff, e.g. parent it's all typed as any by Apollo themselves so I wouldn't waste too much time with that.

I thought the CI wouldn't let us merge this, but it looks like it actually will. So all good.

File: back/polkassembly-front-server/kubernetes/server-namespace.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+kind: Namespace
```
Comment (yjkimjunior) [inline back/polkassembly-front-server/kubernetes/server-namespace.yaml]: don't think this needs to be duplicated here since it's the same namespace.

Comment (Tbaut) [reply back/polkassembly-front-server/kubernetes/server-namespace.yaml]: Not sure I get that. do you mean the same namespace as the `server`?

Comment (yjkimjunior) [reply back/polkassembly-front-server/kubernetes/server-namespace.yaml]: you already have a namespace `polkassembly` created from the Polkassembly repo, so why duplicate it here? ... unless you were trying to create another namespace for the server in which case the below lines should be edited.

Comment (Tbaut) [reply back/polkassembly-front-server/kubernetes/server-namespace.yaml]: Understood thanks, I've no clue what I'm doing when touching kubernetes config files.

**Comment by Tbaut** [comment]
Added motions and tried some nested queries in vain unfortunately. I need to dig a little deeper into this.

**Comment by yjkimjunior** [comment]
I'm fine to merge this here now if it unblocks you guys @niklabh @Tbaut

**Comment by Tbaut** [comment]
Sorry to let this lag behind. 
My idea is to merge this with `server` in the next days, and to move to using it.

It's not extremely urgent since only parity ppl use Polkassembly now, but it'll get to the top soon.

**Comment by Tbaut** [comment]
Here we go, I tested a nested query successfully. Unless I missed anything, this should cover all cases.

We'll move to it as soon as this is done https://github.com/paritytech/polkassembly/issues/372

**Pull request merged**